sealedenumprocessor/src/test/kotlin/com/livefront/sealedenum/compilation/basic/EmptySealedClass.kt

```diff
@@ -29,12 +30,14 @@ public enum class EmptySealedClassEnum
  * The isomorphic [EmptySealedClassEnum] for [this].
  */
 public val EmptySealedClass.`enum`: EmptySealedClassEnum
+    @Suppress("REDUNDANT_MODIFIER_IN_GETTER")
```

Comment:
So this is referring to `public get()`?

Reply:
Correct, this is a compiler warning since `get()` has to match the visibility of the `val`. Once there's a new release of Kotlin Poet we'll be able to remove the suppression.

Reply:
OK sounds good.

Reply:
And that time is now! `1.7.2` was released, I force pushed this branch to have a single commit with the complete update. The update no requires no actual code changes, just expected source updates in tests!